Nosenti/Innocent-Ingabire-portfolio
# Ft ui admin pages

#### What does this PR do?
 Add four Admin UI Pages: Edit, Write, Manage Blogs and Inbox
Validate write blog and subscribe forms
Create, Retrieve and delete blogs from Firebase

#### Description of Task to be completed
 Add four Admin UI Pages: Edit, Write, Manage Blogs and Inbox
Validate write blog and subscribe forms
Create, Retrieve and delete blogs from Firebase
#### How should this be manually tested?
Clone the repository, Open Admin folders and click on any HTML file
Navigate through different pages to see designs
Submit data using contact, subscribe, write blogs form to the database
Check the ManageBlogs' HTML page to see blog posts from Firebase
Delete any blog post of your choice by clicking on the delete button
Open landing pages' HTML file and you will be prompted to accept or decline being tracked by geolocation API

#### Any background context you want to provide?
I am half-way through implementing updating blogs

#### What are the relevant stories?
<blockquote class="trello-card"><a href="https://trello.com/c/uzVcLOBy/20-user-location-using-geolocation-api">User location using geolocation API</a></blockquote><script src="https://p.trellocdn.com/embed.min.js"></script>
<blockquote class="trello-card"><a href="https://trello.com/c/h0aljg1R/23-delete-the-blog">Delete the blog</a></blockquote><script src="https://p.trellocdn.com/embed.min.js"></script>
<blockquote class="trello-card"><a href="https://trello.com/c/uyjXwqOj/24-read-the-blog-from-db">Read the blog from DB</a></blockquote><script src="https://p.trellocdn.com/embed.min.js"></script>
<blockquote class="trello-card"><a href="https://trello.com/c/HhQaTx9v/21-create-a-blog">Create a blog</a></blockquote><script src="https://p.trellocdn.com/embed.min.js"></script>
